packages/example/src/pages/NonVirtualizedGridPage.tsx

```diff
@@ -0,0 +1,66 @@
+import { DefaultFocus, SpatialNavigationNode } from 'react-tv-space-navigation';
+import { Page } from '../components/Page';
+import '../components/configureRemoteControl';
+import { getPrograms } from '../modules/program/infra/programInfos';
+import { useNavigation } from '@react-navigation/native';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { RootStackParamList } from '../../App';
+import styled from '@emotion/native';
+import { Fragment } from 'react';
+import { scaledPixels } from '../design-system/helpers/scaledPixels';
+import { ProgramNode } from '../modules/program/view/ProgramNode';
+
+export const NonVirtualizedGridPage = () => {
+  return (
+    <Page>
+      <Container>
+        <SpatialNavigationNode alignInGrid>
+          <Fragment>
+            <DefaultFocus>
+              <ShortProgramList />
+            </DefaultFocus>
+            <ShortProgramList />
+            <ShortProgramList />
+          </Fragment>
+        </SpatialNavigationNode>
+      </Container>
+    </Page>
+  );
+};
+
+const ShortProgramList = () => {
+  const navigation = useNavigation<NativeStackNavigationProp<RootStackParamList>>();
+  const programs = getPrograms().slice(0, 6);
+  return (
+    <SpatialNavigationNode orientation="horizontal">
+      <ListContainer>
+        {programs.map((program) => {
+          return (
+            <ProgramNode
+              programInfo={program}
+              onSelect={() => navigation.push('ProgramDetail', { programInfo: program })}
+              key={program.id}
+            />
+          );
+        })}
+      </ListContainer>
+    </SpatialNavigationNode>
+  );
+};
+
+const ListContainer = styled.View(({ theme }) => ({
+  flexDirection: 'row',
+  flexWrap: 'wrap',
+  justifyContent: 'center',
+  gap: theme.spacings.$4,
+  padding: theme.spacings.$4,
+}));
+
+const Container = styled.View({
+  backgroundColor: '#222',
```

Comment:
Do we have this color in the theme? That would be better to get it from there 😊

Reply:
I'll look for it, I saw it written this way elsewhere so I pasted it but I'll make it better for both here and the place I found it :)